debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -564,29 +538,21 @@ protected String convertSetValue(Column column, long indexes, List<String> optio
      * @throws IllegalArgumentException if the value could not be converted but the column does not allow nulls
      */
     protected Object convertPoint(Column column, Field fieldDefn, Object data){
-        if (data == null) {
-            data = fieldDefn.schema().defaultValue();
-        }
-
-        Schema schema = fieldDefn.schema();
-
-        if (data == null) {
-            if (column.isOptional()) return null;
+        return convertValue(column, fieldDefn, data, (Supplier<?>)() -> {
             // we can't create an EMPTY Point because it has X & Y integer fields which can't be null.
             throw new IllegalArgumentException("Nulls not valid on " + column);
```

Comment:
Can't we just pass null as `fallback` and raise the exception in `convertValue()` when we would want to access it (no value given for non-optional column without schema value)?

Reply:
I'll pass the same eempty value like oin geomtry. It is a change in semnatics but it has no impact on backward compatibility I'd say.